src/test_l1_dsVideoDevice.c

```diff
@@ -166,7 +166,11 @@ void test_l1_dsVideoDevice_negative_dsVideoDeviceInit(void)
 
     // Step 02: Try initializing again without terminating
     result = dsVideoDeviceInit();
-    DS_ASSERT_AUTO_TERM_NUMERICAL(result, dsERR_ALREADY_INITIALIZED);
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
https://github.com/rdkcentral/ut-core/discussions/55 -> please update to support the KVP module , so we can commit this code as a valid upgrade, that can change configuration at runtime.